src/FileList/FileExtInfoLoader.cc

```diff
@@ -151,9 +152,14 @@ bool FileExtInfoLoader::FillFileInfoFromImage(CARTA::FileInfoExtended& extended_
                 // Create header entry for each FitsKeyword
                 fhi.kw.first(); // go to first card
                 casacore::FitsKeyword* fkw = fhi.kw.next();
+                std::set<casacore::String> name_set; // used to check the repetition of name
+                casacore::Regex underscore_suffix(casacore::Regex::fromPattern("*_"));
```

Comment:
why can't we use `std::regex` here? We know there are issues with `casacore::Regex`